### ISSUE TITLE
Publish Busybox-w32 on winget

### ISSUE THREAD

**Issue by hummeleBop** [rmyorston/busybox-w32#272]
I would like to request that Busybox be published on winget. It will be easier for users to find and install the software if it is available through winget.

There are a few advantages of publishing a software on winget: 

1. It is a great way to get your software in front of a large audience of potential users. 
2. It can help increase the visibility of your software and help you to reach a wider audience. 
3. It can also help to increase the search engine ranking of your software, making it easier for potential users to find it.

Thanks a lot :)

**Comment by rmyorston** [comment]
This isn't something I'm interested in doing myself.

A cursory reading of the winget documentation suggests they need software to be packaged in an installer, which busybox-w32 isn't.

As an aside, busybox-w32 is already available via [Scoop](https://scoop.sh) and [Chocolatey](https://community.chocolatey.org/packages/busybox)

**Comment by hummeleBop** [comment]
I agree with you.

The problem about packaging the application in a zip file has been solved in the latest update (currently in a preview version though). Since 1.4, you can package the application as a zip file.

If you're okay with it, I'm interested in helping to develop the winget zip package.

**Comment by rmyorston** [comment]
@hummeleBop I'm happy to co-operate on getting a winget package put together.

Though since you seem to know what you're talking about you get to do all the work!

**Comment by bsdayo** [comment]
The winget v1.4 containing .zip package support is released, should this issue be picked up again?

It will be amazing if we can directly install busybox via winget :)

**Comment by Kreijstal** [comment]
could you do this on the msys2 repository?

**Comment by rmyorston** [comment]
@Kreijstal I'm not clear what you mean. msys2 seems to have its own BusyBox package.

**Comment by Kreijstal** [comment]
> @Kreijstal I'm not clear what you mean. msys2 seems to have its own BusyBox package.

it doesnt have that drop privilges command :(, could it be ported?

**Comment by rmyorston** [comment]
I'm afraid not. MSYS2 takes a different approach to porting to Windows. It uses a modified version of Cygwin to allow Unix applications to built on Windows.

The BusyBox in MSYS2 is their build of the upstream code without any of the Windows-specific features in busybox-w32.

**Comment by Kreijstal** [comment]
yeah, but they still cross-compile from msys(cygwin) to windows (without posix features) using mingw, from msys2 itself. So you still have access to mingw and all of user32 userland, right?
